### ISSUE TITLE
where is the 32 bit software?

### ISSUE THREAD

**Issue by AhMeD-PS4** [BtbN/FFmpeg-Builds#108]
i was searching for a 32 bit build but, there isn't?, even old?
i searched for and old build but i was ended by the page 3 or 4
if you have a 32 bit build pls gimme it

**Comment by sudo-nautilus** [comment]
No, they aren't broken. But by seeing the same, people *may* think that you're also providing x86 daily autobuilds too

**Comment by BtbN** [comment]
Given the amount of Pull Requests i close on the main FFmpeg Github Mirror, I'm not convinced people will read that either :D

**Comment by sudo-nautilus** [comment]
Ah I see :P

**Comment by pzhlkj6612** [comment]
> Given the amount of Pull Requests i close on the main FFmpeg Github Mirror, I'm not convinced people will read that either :D

OK, you're right.

Maybe the obsolescence of 32-bit operating systems is what we really need...
